Pipfile

```diff
@@ -26,7 +26,7 @@ python_version = "3.8"
 [scripts]
 lint = "pre-commit run --all-files"
 precommit = "pre-commit install"
-autoformat = "black ."
+autoformat = "black src"
```

Comment:
Not sure this is a good idea, we might have some python files outside `src/` which also needs to be formatted with `black`